pkgpanda/docs/http-swagger.yaml

```diff
@@ -4,6 +4,11 @@ info:
   version: '1.0.0'
 basePath: /pkgpanda
 
+tags:
+  - name: repository
+    description: manage installed packages
+  - name: active
+    description: manage activated packages
```

Comment:
well, it allows you to activate packages that are installed... imo the top level primitives are poorly named, but yeah I can change it...